src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -49,11 +49,20 @@
 from psyclone.transformations import ACCLoopTrans, ACCEnterDataTrans, \
     ACCParallelTrans, ACCKernelsTrans
 from psyclone.tests.utilities import get_invoke
+from psyclone.configuration import Config
```

Comment:
Could you sort the import alphabetically, please?

Reply:
Done